### DISCUSSION TITLE
How to set the mysql timezone?

### DISCUSSION THREAD

**Discussion by Kamaruzaman** [planetscale/discussion#297] [Q&A] [answered]
Hello,

How to change MySQL timezone? since my application uses GMT+8 as the default timezone, all of the previous records are different compared with a new timezone.

**Comment by fooforge** [comment, marked as answer]
@Kamaruzaman Hi 👋,

Databases hosted on PlanetScale use UTC as their global timezone and you will not be able to change it. We do, however, have Timezone support enabled:

```sql
database/branch> SELECT COUNT(*) FROM mysql.time_zone_name;
+----------+
| count(*) |
+----------+
|     1826 |
+----------+
```

This allows you to use MySQL's [date and time functions](https://dev.mysql.com/doc/refman/8.0/en/date-and-time-functions.html) such as `CONVERT_TZ`:

```sql
database/branch> SELECT
    ->   CONVERT_TZ('2007-03-11 2:00:00','US/Eastern','US/Central') AS time1,
    ->   CONVERT_TZ('2007-03-11 3:00:00','US/Eastern','US/Central') AS time2;
+---------------------+---------------------+
| time1               | time2               |
+---------------------+---------------------+
| 2007-03-11 01:00:00 | 2007-03-11 01:00:00 |
+---------------------+---------------------+
```

Generally, my recommendation would be to handle date/time on the application level as it gives you more control.
